app-connector-client/server/config.js

```diff
@@ -56,10 +57,26 @@ function configValidation(configJson) {
             }
         }
     }
+    if (apis) {
+        for (var i = 1; i <= apis.length; i++) {
+            {
+                var api = apis[i - 1];
+
+                if (!api.authtype) {
```

Comment:
"authtype" does not look nice... what about simple "auth"?

Reply:
changed to auth